Verify.Nupkg.sln

```diff
@@ -0,0 +1,44 @@
+﻿
```

Comment:
Thanks for this!

sln files tend to bother me because they don't compose well and are filled with GUID goop. On bigger projects I use [slngen](https://github.com/microsoft/slngen).

However, I don't expect the project structure to change much for this repo, so a .sln is much more approachable!